cellengine/resources/gate.py

```diff
@@ -162,81 +144,347 @@ def create(
         x2: float,
         y1: float,
         y2: float,
-        label: List[str] = [],
-        gid: str = None,
+        label: List[float] = [],
+        gid: Optional[str] = None,
         locked: bool = False,
-        parent_population_id: str = None,
-        parent_population: str = None,
         tailored_per_file: bool = False,
-        fcs_file_id: str = None,
-        fcs_file: str = None,
+        fcs_file_id: Optional[str] = None,
+        fcs_file: Optional[str] = None,
         create_population: bool = True,
-    ) -> RectangleGate:
-        g = format_rectangle_gate(**get_args_as_kwargs(cls, locals()))
-        return cls(g)
+        parent_population_id: str = None,
```

Comment:
They're popped from the kwargs, because they aren't gate properties, but they're passed to `APIClient.create()` as [query parameters](https://docs.cellengine.com/api/#create_3)